packages/components/src/internal/components/forms/QueryFormInputs.tsx

```diff
@@ -209,7 +211,7 @@ export class QueryFormInputs extends React.Component<QueryFormInputsProps, State
                                 data={fieldValues}
                                 formsy
                                 initiallyDisabled={shouldDisableField}
-                                key={i}
+                                key={col.name}
```

Comment:
I'm not sure we're guaranteed for `col.name` to be unique as there may be lookup fields from other domains that have the same `name`. I would consider `col.fieldKey` (which even that I'm not sure of for the same reason).

Reply:
We can't depend on fieldKey being unique? That seems bad.

Reply:
I reverted the key changes